fearless/Modules/Root/RootInteractor.swift

```diff
@@ -78,5 +78,19 @@ extension RootInteractor: RootInteractorInputProtocol {
     func setup() {
         setupURLHandlingService()
         runMigrators()
+
+        // TODO: Move to loading screen
+        settings.setup(runningCompletionIn: .main) { result in
+            switch result {
+            case let .success(maybeMetaAccount):
+                if let metaAccount = maybeMetaAccount {
+                    self.logger?.debug("Selected account: \(metaAccount.metaId)")
+                } else {
+                    self.logger?.debug("No selected account")
+                }
+            case let .failure(error):
+                self.logger?.error("Selected account setup failed: \(error)")
```

Comment:
Do we only log the result here? Are there any other actions?

Reply:
I think we could add error state but this case can happen when something broken in the database. As we assume that a user will not change db manually then it is ok for now just log the error and show onboarding screen.